src/crypto/PrivateKeyBundle.ts

```diff
@@ -3,7 +3,6 @@ import { PrivateKey, SignedPrivateKey } from './PrivateKey'
 import { WalletSigner } from './Signature'
 import { PublicKey, SignedPublicKey } from './PublicKey'
 import { PublicKeyBundle, SignedPublicKeyBundle } from './PublicKeyBundle'
-import { Signer } from 'ethers'
```

Comment:
I don't love the fact that we are exporting this type globally. Feels safer and easier to understand to actually import the type upon usage. Right now it just appears like magic, which is going to cause developer confusion.

I also don't remember where we configured this magic, so that things from the `types` folder were available in every module. I think it makes sense for something like a polyfill, but less so for an application-specific type like this.

Reply:
> I don't think this is special to the types folder - if you don't export or import from within any ts file it becomes global.

How did I not know that?